DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Receive/DisableRedrawScope.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Rhino.DocObjects.Tables;
+
+namespace Speckle.Connectors.Rhino7.Operations.Receive;
+
+/// <summary>
+/// Helper class to disable <see cref="ViewTable.RedrawEnabled"/> within a scope
+/// </summary>
+public sealed class DisableRedrawScope : IDisposable
+{
+  private readonly ViewTable _viewTable;
+  private readonly bool _returnToStatus;
+
+  public DisableRedrawScope(ViewTable viewTable, bool returnToStatus = true)
+  {
+    _viewTable = viewTable;
+    _returnToStatus = returnToStatus;
+
+    _viewTable.RedrawEnabled = false;
+  }
+
+  public void Dispose()
+  {
+    _viewTable.RedrawEnabled = _returnToStatus;
+  }
```

Comment:
Do we need return to status if this class explicitly only Disables the Redraw scope? (i.e. it won't ever switch it "on", making the need of returning to the prior status somewhat redundant?)

Reply:
No, I was thinking that, if we start using this in a nested way, you'd actually want to disable, then restore to what ever the status was before the scoped block, that would prevent us from ever accidentally enabling on dispose when we should actually stay disabled.

I thought to kepp it simple for this pr, do that parameter is always true

If you think what I describe is not useful, we can simply remove it from this class

Reply:
No this is fine. I was just wondering since you weren't using it currently; but I agree it is a good idea to have it.